src/zkevm_specs/evm/execution/end_block.py

```diff
@@ -59,6 +71,7 @@ def end_block(instruction: Instruction):
     total_rws = (1 - is_empty_block) * (instruction.curr.rw_counter - 1 + 1)
```

Comment:
```suggestion
    total_rws = (1 - is_empty_block) * (instruction.curr.rw_counter - 1 + 2)
```
Since now we're doing an extra rw_lookup (for the TxReceiptFieldTag.CumulativeGasUsed) I think this should be updated.  And the comment as well, it should mention "1 call_context and 1 tx_receipt lookup".